docs/milo/manual/chapters/90_install_rrf.md

```diff
@@ -108,39 +108,39 @@ You should see a _green_ highlighted line indicating that the command was succes
 
 You can also [connect to the mainboard over USB](https://teamgloomy.github.io/putty.html) using Putty or another terminal emulator depending on your operating system - watching the output of the terminal when the machine connects to the WiFi network will show the IP address that has been assigned.
 
 !!! warning
     When installing a new release of the RRF configuration as an _upgrade_, your `network.g` file will not be overwritten. _However_ - the WiFi module will be updated, which may wipe all network details from the module itself.
 
-    To avoid this making the machine unrecoverable (at least without connecting to USB), RRF will always reboot in Access Point mode after any update cycle. You can then re-enter the network details if necessary, using the `M587` command in [Configure your WiFi network](#configure-your-wifi-network) before rebooting.
+    To avoid making the machine unrecoverable (at least without connecting to USB), RRF will always reboot in Access Point mode after any update cycle. You can then re-enter the network details if necessary, using the `M587` command in [Configure your WiFi network](#configure-your-wifi-network) before rebooting.
 
     When you are prompted by DWC to Install the updates, you should click Yes - but be aware, this will involve flashing your WiFi module twice - once when you click Yes, and once when the board reboots after it is updated. This is unfortunate, but necessary, because we need the flash-on-reboot to enable hands-free configuration when extracting the Zip file to a fresh SD card and inserting it into an unconfigured board.
 
 ---
 
-At this point, the basic firmware setup is complete and all that remains are some firmware checks which will be covered later on in the manual.
+At this point, the basic firmware setup is complete and all that remains are some firmware checks which will be covered later in the manual.
 
 !!! warning
-    We only provide bundled firmware configurations for specific machine setups. If your machine is self-built, you will have to modify one of these configurations to match the mainboard, wiring and setup of your machine. Alternatively, you can use the [TeamGloomy Configurator](https://teamgloomy.github.io/Configurator) - but remember that this is aimed at 3D printer configurations so will need a number of changes to work effectively on Milo.
+    We only provide bundled firmware configurations for specific machine setups. If your machine is self-built, you have to modify one of these configurations to match the mainboard, wiring, and setup of your machine. Alternatively, you can use the [TeamGloomy Configurator](https://teamgloomy.github.io/Configurator) - but remember that this is aimed at 3D printer configurations so it will need several changes to work effectively on Milo.
```

Comment:
I'm not sure I like the change to several here. Several sounds like we know how many changes are required, but the real answer is we don't because it'll be different based on what the user picks in the configurator and their specific machine setup.

Maybe we go with something like "so it will need changing to suit the particular features of your Milo".